Common.props

```diff
@@ -1,11 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), 'Global.props'))\Global.props" />
-  <Import Project=".\GlobalStaticVersion.props" />
```

Comment:
I think that might be necessary.

The issue is that if the property is set after Microsoft.Common.props has been imported, then it is too late for it to pick up the .props files generated in that folder by NuGet restore.  You can resolve this by always importing Global.props file before Microsoft.Common.props gets imported, but I found it tricky to ensure. An easier way, and this is what I have done here, is to use the special Directory.Build.props file which will automatically get imported beforehand by MSBuild.  For more information, see https://github.com/Microsoft/msbuild/issues/1603#issuecomment-283104367